extensions/ql-vscode/package.json

```diff
@@ -198,6 +198,11 @@
             "maximum": 1024,
             "description": "Number of threads for running queries."
           },
+          "codeQL.runningQueries.ignoredFolders": {
```

Comment:
I suggest we rename to `codeQL.runningQueries.ignoredWorkspaceFolders`. It's a bit longer but it makes it clear just from the name that it only applies to workspace folders and not to arbitrary folders.